src/middlewares/tripMiddleware.js

```diff
@@ -0,0 +1,20 @@
+import { findTripById } from '../services/tripServices';
+import { respondWithWarning } from '../helpers/responseHandler';
+import statusCode from '../helpers/statusCode';
+import resMessage from '../helpers/responseMessages';
+
+export const isTripExist = async (req, res, next) => {
```

Comment:
`isTripExist ` could be renamed to `verifyTrip`

Reply:
ok.Thanks for the feedback. I will do that now.